src/v0/shift_plan.ts

```diff
@@ -62,22 +62,22 @@ export class ShiftPlan extends ThBaseHandler {
 
       return {
         msg: response.data.msg,
-        data: response.data.results[0],
+        data: response.data.results,
```

Comment:
This is a `get` method which is supposed to return a single Shift Plan, however, a shift plan is composed of multiple shifts ( its an array ), is it fine for the `get` method to return an array?

Had some issues with the use-model-new hook because when it sets the data in the model, it uses `Object.keys` to clone it, but since this is an array it didnt work and I had to fix it through parsing the results and returning an object

![image](https://github.com/tillhub/tillhub-sdk-javascript/assets/140511059/7a420ef2-eae5-442c-8b6e-d205c28e0bfd)

Reply:
> is it fine for the get method to return an array?
it's not fine but.... it's not unique for our API

Reply:
So, should I leave it like this?

Reply:
Yes, let's leave it like this